x-pack/legacy/plugins/apm/cypress/support/commands.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// ***********************************************
```

Comment:
Maybe we can remove this, feels like noise.

Reply:
True. This is some files that cypress generates automatically on `yarn install` if they don't exist. So I opted just to leave them as-is.

I'll remove the commented out code.